src/components/header/HeaderComponent.vue

```diff
@@ -291,3 +291,9 @@ const badgeCss = 'fr-badge fr-badge--sm fr-badge--' + props.badgeStyle
   gap: 0.25em;
 }
 </style>
+
+<style>
+.fr-header__tools-links .fr-btns-group li .fr-btn {
```

Comment:
```suggestion
.fr-header__tools-links :deep(.fr-btn) {
```

Ça fonctionnera dans le `scoped` comme ça !

Reply:
Le  hover/active est un peu bizarre sans padding du coup

![image](https://github.com/user-attachments/assets/d4740d7a-b535-4116-90c8-c1bc9d70b334)

Reply:
Ah bien vu ! Je retire